projects/packages/my-jetpack/_inc/components/stats-section/index.jsx

```diff
@@ -11,7 +11,8 @@ const StatsSection = () => {
 	const slug = 'stats';
 	const { detail } = useProduct( slug );
 	const { status } = detail;
-	const { userIsAdmin } = window?.myJetpackInitialState ?? false;
```

Comment:
The value of `userIsAdmin` in the initial state is `'1'`.
<img width="322" alt="Screenshot 2024-02-20 at 11 23 53 AM" src="https://github.com/Automattic/jetpack/assets/1620183/f69a69b3-fb42-40f6-bdbf-2a32927d7b3b">